File: token-validation-spring-test/src/test/java/no/nav/security/token/support/spring/test/TestApplication.java
package no.nav.security.token.support.spring.test;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.Configuration;

@Configuration
@EnableAutoConfiguration
class TestApplication {
    public static void main(String[] args) {
        SpringApplication.run(TestApplication.class, args);
    }
}

File: token-validation-spring-test/src/test/kotlin/no/nav/security/token/support/spring/test/TestApplication.kt
package no.nav.security.token.support.spring.test

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Configuration

@Configuration
@EnableAutoConfiguration
class TestApplication {

    fun main(args : Array<String>) {
        runApplication<TestApplication>(*args)
    }
}